{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure_basic.py:16: Exception"}, "start": 1592303807802, "stop": 1592303807802, "uuid": "d4355c11-e78e-42f3-9786-3d5f94496775", "historyId": "cdce425562f54e2475d3dc800a45dbe3", "testCaseId": "e6df84670fcda2692a7b44adf6013d43", "fullName": "test_allure_basic#test_broken", "labels": [{"name": "suite", "value": "test_allure_basic"}, {"name": "host", "value": "LAPTOP-4F3KR3S3"}, {"name": "thread", "value": "13920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure_basic"}]}